.github/workflows/update.yaml

```
@@ -81,6 +81,9 @@ jobs:
           # Convert addon strings to XLIFF
           addon_files="vpn/addon_ts/*.ts"
 
+          # Copy the extras.xliff file over
+          cp vpn/src/translations/extras/extras.xliff translationFiles/en/extras.xliff
```

I thought about ignoring failure here until the MozVPN PR... But I'd rather block merging this on merging the MozVPN Pr instead. It will probably get merged today anyways.

---

```suggestion
          if [ -d "vpn/src/translations/extras/extras.xliff" ]; then
            cp vpn/src/translations/extras/extras.xliff translationFiles/en/extras.xliff
          fi          
```
We need to guard against failure anyway, because we're also looking at release branches that won't have this folder for a while.

---

We should also move this a couple of lines above (before `# Convert addon strings to XLIFF`).